### PR TITLE
Install Tk private headers

### PR DESCRIPTION
**Pull request by hajgato**
See http://www.linuxfromscratch.org/blfs/view/8.2/general/tk.html

### CODE REVIEW

File: easybuild/easyconfigs/t/Tk/Tk-8.6.8-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -26,7 +26,10 @@ dependencies = [
 
 configopts = '--enable-threads --with-tcl=$EBROOTTCL/lib CFLAGS="-I$EBROOTTCL/include"'
 
-postinstallcmds = ["ln -s wish%(version_major_minor)s %(installdir)s/bin/wish"]
+postinstallcmds = [
+    "make install-private-headers",
```
Comment (boegel) [inline easybuild/easyconfigs/t/Tk/Tk-8.6.8-GCCcore-6.4.0.eb]: @hajgato Wouldn't it be better to do this via `installopts`?

```python
installopts = "&& make install-private-headers"
```

Also, we should enhance `sanity_check_paths` to check for at least one of these header files?

Comment (hajgato) [reply easybuild/easyconfigs/t/Tk/Tk-8.6.8-GCCcore-6.4.0.eb]: @boegel Done.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2606.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/44517ac38fb83654a5d46cfc55b74ced for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2040.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/801c8d775544f584629d694719b08515 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (4 easyconfigs in this PR)
node3124.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/6cd8d529e712fe0ccd9426727da78b39 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**